libs/full/program_options/docs/index.rst

```diff
@@ -12,8 +12,8 @@ program_options
 ===============
 
 The module program_options is a direct fork of the Boost.ProgramOptions library
-(Boost V1.70.0). For more information about this library please see `here
-<https://www.boost.org/doc/libs/1_70_0/doc/html/program_options.html>`__.
+(Boost V1.71.0). For more information about this library please see `here
+<https://www.boost.org/doc/libs/1_71_0/doc/html/program_options.html>`__.
```

Comment:
Hmm, it is still based on 1.70 even if we require 1.71. I'd either keep the note about 1.70, or remove it altogether.

Reply:
I'd like to see this note left in place (while referring to the original V1.70.0).